src/hydra/parseHydraDocumentation.js

```diff
@@ -7,6 +7,16 @@ import Operation from "../Operation";
 import fetchJsonLd from "./fetchJsonLd";
 import getParameters from "./getParameters";
 
+/**
+ * Extracts the url of a resource.
+ *
+ * @param {string} propertyId
+ * @return {string}
+ */
+function guessUrlFromPropertyId(propertyId) {
+  return `${propertyId.replace("docs.jsonld#Entrypoint/", "")}s`;
```

Comment:
This hardcoded replacement creates a hard coupling with API Platform Core while we want this lib to work with any Hydra implementation.

The base URL can be extracted from `hydra:entrypoint` (and is probably available somewhere as a var).

The resource name could be extracted using the `URL` API (hash part) and a regex, even.

Some plural don't end by a `s`, this shouldn't be hardcoded as well.